python/ql/lib/semmle/python/RegexTreeView.qll

```diff
@@ -535,6 +536,13 @@ private int toHex(string hex) {
   result = 15 and hex = ["f", "F"]
 }
 
+/**
+ * A word boundary, that is, a regular expression term of the form `\b`.
+ */
+class RegExpWordBoundary extends RegExpEscape {
```

Comment:
I'm not convinced `\b` should be parsed as `RegExpEscape`, because it can result in ReDoS query FPs that say "...strings starting with 'b'...", but I can handle that in #7120.

Reply:
I see.  
Yes, that causes FPs in Ruby/Python (I checked). 

Nice if you can handle it in #7120.